x/stakeibc/keeper/transfer.go

```diff
@@ -109,13 +60,13 @@ func (k Keeper) TransferCommunityPoolDepositToHolding(ctx sdk.Context, hostZone
 		hostZone.ConnectionId,
 		msgs,
 		types.ICAAccountType_COMMUNITY_POOL_DEPOSIT,
-		icaTimeoutTimestamp,
+		endEpochTimestamp,
 		icaCallbackId,
 		icaCallbackData)
 	if err != nil {
 		return errorsmod.Wrapf(err, "Failed to SubmitTxs, Messages: %v, err: %s", msgs, err.Error())
 	}
-	k.Logger(ctx).Info("[TransferCommunityPoolTokens] Successfully sent ICA command to kick off remote ibc transfer from community pool deposit ICA")
+	k.Logger(ctx).Info("Successfully sent ICA command to kick off ibc transfer from deposit ICA to holding address")
```

Comment:
ty for removing the [FUNCTION NAME], we should get rid of that across the codebase too